monkey/monkey_island/cc/ui/src/components/ui-components/plugins-marketplace/AvailablePlugins.tsx

```diff
@@ -182,33 +183,38 @@ const AvailablePlugins = (props) => {
 
   return (
     <Stack spacing={2} height='100%' id={styles['available-plugins']}>
-      <Grid container spacing={2}>
-        <Grid xs={4} item
-              sx={{alignItems: 'flex-end', display: 'flex'}}>
-          <SearchBar setQuery={onSearchChanged} />
-        </Grid>
-        <Grid xs={3} item >
-          <TypeFilter allPlugins={availablePlugins}
-                      filters={filters}
-                      setFilters={setFilters}
-                      className={'type-filter-box'}/>
-        </Grid>
-        <Grid xs={1} item/>
-        <Grid xs={3} item>
-          <Box display="flex" justifyContent="flex-end" >
-            <Button onClick={installAllSafePlugins} disabled={installingAllSafePlugins}>
-              <FileDownloadIcon/> All Safe Plugins
-            </Button>
-          </Box>
-        </Grid>
-        <Grid xs={1} item >
-          <Button onClick={refreshPlugins}><RefreshIcon className={`${isSpinning && 'spinning-icon'}`}/></Button>
-        </Grid>
-      </Grid>
+      {availablePlugins?.length > 0 && (
```

Comment:
Why are we checking this?

Reply:
to hide the filters if there aren't any plugins to actually filter on

Reply:
Maybe we should move all this JSX into a method called ` TableControls`. This would make the logic clearer, conditional rendering is not meant for 30+ line blocks